django/thunderstore/api/cyberstorm/views/dynamic_html.py

```diff
@@ -0,0 +1,32 @@
+from django.http import Http404
+from django.utils.html import escapejs
+from rest_framework import serializers
+from rest_framework.generics import RetrieveAPIView
+
+from thunderstore.api.utils import CyberstormAutoSchemaMixin
+from thunderstore.frontend.models import DynamicHTML, DynamicPlacement
+
+
+class CyberstormDynamicHTMLResponseSerializer(serializers.Serializer):
+    dynamic_htmls = serializers.ListField(child=serializers.CharField())
+
+
+class DynamicHTMLAPIView(CyberstormAutoSchemaMixin, RetrieveAPIView):
```

Comment:
There's no reason to use the `RetrieveAPIView` base class when you're not returning a db model instance. Use the `GenericAPIView` instead and just implement the `get` method on it.